languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/TempClient.java

```diff
@@ -1,8 +1,127 @@
 package client;
 
+import json.CommercetoolsJsonUtils;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+
 public class TempClient implements ApiHttpClient {
+    private static final boolean LOG = true;
+    
+    private static final String AUTH_EUROPE_BASE_URL = "https://auth.sphere.io";
+    private static final String AUTH_US_BASE_URL = "https://auth.commercetools.co";
+    private static final String API_EUROPE_BASE_URL = "https://api.sphere.io";
+    private static final String AUTH_PATH = "/oauth/token";
+    private static final String INTROSPECT_PATH = "/oauth/introspect";
+    
+    private static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+    private static final OkHttpClient client = new OkHttpClient();
+    private String token = "";
+    
+    public ApiHttpResponse execute(final ApiHttpRequest apiHttpRequest) throws IOException {
+        if(token.isEmpty() || !isTokenActive()){
+            token = obtainAccessToken();
+        }
+        //set path
+        Request.Builder httpRequestBuilder = new Request.Builder()
+                .url(API_EUROPE_BASE_URL + apiHttpRequest.getPath());
+        
+        //set headers
+        for(Map.Entry<String, String> entry : apiHttpRequest.getHeaders().getHeaders().entrySet()){
+            httpRequestBuilder = httpRequestBuilder.header(entry.getKey(), entry.getValue());
+        }
+        httpRequestBuilder.header("Authorization", " Bearer " + token);
+        
+        //set method and body
+        switch (apiHttpRequest.getMethod()){
+            case GET:
+                httpRequestBuilder = httpRequestBuilder.get();
+                break;
+            case DELETE:
+                httpRequestBuilder = httpRequestBuilder.delete();
+                break;
+            case POST:
+                httpRequestBuilder = httpRequestBuilder.post(RequestBody.create(apiHttpRequest.getBody(), JSON));
+                break;
+            case PUT:
+                httpRequestBuilder = httpRequestBuilder.put(RequestBody.create(apiHttpRequest.getBody(), JSON));
+                break;
+            default:
+                throw new RuntimeException("Non supported HTTP Method : " + apiHttpRequest.getMethod().toString());
+        }
+
+        print("<====NEW REQUEST====>");
```

Comment:
We should later replace this with proper logging.

Reply:
yeah, I agree, this is just for debugging while we are still in "alpha" version.